game.js

```diff
@@ -0,0 +1,74 @@
+function getComputerChoice(){
+    let options = ["Rock", "Paper", "Scissors"];
+    const index = Math.floor(Math.random()*options.length);
+    return options[index];
+}
+function playRound(playerSelection,computerSelection) {
+    if (playerSelection.toLowerCase() === computerSelection.toLowerCase()){
```

Comment:
Is it possible to use switch case statement here ?, if-else ladder is never recommended approach until there is no any other way to do this, because `if-else` ladder decrease the coding readability